gcalcli/validators.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import absolute_import
+
+import re
+
+from builtins import input
```

Comment:
python2 doesn't have the `builtins` module

Reply:
I never use Python 2 and have never developed for it, so I'm sorry this slipped through. I've replaces `from builtins import input` with `from six.moves import input`.